src/components/ProductVersionCreateEditPage/ProductVersionCreateEditPage.tsx

```diff
@@ -0,0 +1,199 @@
+import { ActionGroup, Button, Form, FormGroup, FormHelperText, TextInput } from '@patternfly/react-core';
+import { useEffect } from 'react';
+import { useNavigate, useParams } from 'react-router-dom';
+
+import { PageTitles } from 'common/constants';
+import { productVersionEntityAttributes } from 'common/productVersionEntityAttributes';
+
+import { IFieldValues, useForm } from 'hooks/useForm';
+import { useServiceContainer } from 'hooks/useServiceContainer';
+import { useTitle } from 'hooks/useTitle';
+
+import { ContentBox } from 'components/ContentBox/ContentBox';
+import { PageLayout } from 'components/PageLayout/PageLayout';
+import { ServiceContainerCreatingUpdating } from 'components/ServiceContainers/ServiceContainerCreatingUpdating';
+
+import * as productVersionApi from 'services/productVersionApi';
+
+import { validateProductVersionName } from 'utils/formValidationHelpers';
+import { createSafePatch } from 'utils/patchHelper';
+import { generatePageTitle } from 'utils/titleHelper';
+
+const formConfig = {
```

Comment:
Just a reminder we started to use updated name with type

Reply:
I will rebase the PR and update accordingly.